### ISSUE TITLE
Bug: Capitalization in file name causing 404s

### ISSUE THREAD

**Issue by DavidWells** [MoOx/phenomic#832]
It looks like when we publish a blog post with a capital letter the page 404s

Live example: https://58090f7f71e20a185030a3f5--serverless.netlify.com/blog/Serverless-post-1.0/ if you visit this URL it will show then 404.

```
// in console
PageContainer: /blog/serverless-post-1.0/ is a page not found.
```

If you visit the same URL via react router from https://58090f7f71e20a185030a3f5--serverless.netlify.com/blog then click on the link, the page doesn't 404.

Is this a problem with react router routes or the dist folder names?

![image](https://cloud.githubusercontent.com/assets/532272/19576319/6bdbb2ba-96c7-11e6-9224-a02fc1f128ae.png)

**Comment by MoOx** [comment]
I guess it's a bug from phenomic code, but not sure why yet. Maybe in the loader. I don't have for this today, maybe this evening but not even sure.

**Comment by DavidWells** [comment]
No rush. We can 'fix' it on our side by keeping file names lower case.

Other people will probably run into this eventually too =P

**Comment by MoOx** [comment]
I just added some tests to check if there is something to do with the server or PageContainer.

The loader is fine

![image](https://cloud.githubusercontent.com/assets/157534/19609176/fc23b048-97d5-11e6-831d-d36579b4430f.png)

![image](https://cloud.githubusercontent.com/assets/157534/19609182/05e28e6a-97d6-11e6-912f-d0c0312bdeb8.png)

The dev server is fine too

![image](https://cloud.githubusercontent.com/assets/157534/19609190/146f2e0c-97d6-11e6-9f60-64aa34a4c444.png)

![image](https://cloud.githubusercontent.com/assets/157534/19609196/1c86316c-97d6-11e6-9937-c77aed29e9f8.png)

![image](https://cloud.githubusercontent.com/assets/157534/19609169/ef9719c8-97d5-11e6-9748-2384ec924f7c.png)

And the build is also fine

![image](https://cloud.githubusercontent.com/assets/157534/19609228/409739b6-97d6-11e6-94cb-3108932b6458.png)

Can you check your build folder output?
Also can you check it's not related to your server?

I don't see what can cause this issue from phenomic code :/

**Comment by DavidWells** [comment]
The build works and the url works when react-router { Link } is clicked to the camelCased url.

What doesn't work is direct links from external sites and hard page loads (https://58090f7f71e20a185030a3f5--serverless.netlify.com/blog) 

See this in action: http://www.screencast.com/t/9comvZJKi

This isn't a huge deal (b/c you can solve by making sure all your files are named with lowercase only) but might be something people run into.

**Comment by bloodyowl** [comment]
Going to be fixed with #925

**Comment by MoOx** [comment]
Probably not relevant anymore with v1.
